### PR TITLE
List component added 

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
### Folder Structure:
Component Folder
Drawer Folder
Images Folder
document.css
document.html
index.css
index.html
main.css

Go to the the component folder to get the CSS of all the individual components.
HTML of all the components is to be found in document.html ( scroll down to get the relevant component)

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    font-family:'Courier New', Courier, monospace
+}
+
+.list-item{
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  
+}
+
```
Comment (anuj-kumary) [inline components/list.css]: format document remove this spaces

Comment (Shah-Arsalan) [reply components/list.css]: Done . Can you approve now?

**Review by anuj-kumary — APPROVED** [review]

File: document.html [hunk 2]
```diff
@@ -608,6 +609,74 @@ <h2 class="title-prominent">San Francisco, CA</h2>
...
+                    </div>
+
+                    <div class="radio-list-items">
+                        <div class="list-item">
+                            <input id="item-1" type="radio" name="radio-input">
+                            <label for="item-1">item 1</label>
+                        </div>
+                        <div class="list-item">
+                            <input id="item-2" type="radio" name="radio-input">
+                            <label for="item-1">item 1</label>
```
Comment (Rohanmond) [inline document.html:631]: put item-2 in for

File: document.html [hunk 3]
```diff
@@ -608,6 +609,74 @@ <h2 class="title-prominent">San Francisco, CA</h2>
...
+                        </div>
+
+                        <div class="list-item">
+                            <input id="item-3" type="radio" name="radio-input">
+                            <label for="item-2">item 1</label>
```
Comment (Rohanmond) [inline document.html:636]: here also put item-3 inside for

File: document.html [hunk 4]
```diff
@@ -608,6 +609,74 @@ <h2 class="title-prominent">San Francisco, CA</h2>
...
+                    </div>
+
+                    <div class="checkbox-list-items">
+                        <div class="list-item">
+                            <input id="item-1" type="checkbox" name="checkbox-input">
+                            <label for="item-1">item 1</label>
+                        </div>
+                        <div class="list-item">
+                            <input id="item-2" type="checkbox" name="checkbox-input">
+                            <label for="item-1">item 1</label>
```
Comment (Rohanmond) [inline document.html:661]: here also put item-2 inplace of item-1 inside label

File: document.html [hunk 5]
```diff
@@ -608,6 +609,74 @@ <h2 class="title-prominent">San Francisco, CA</h2>
...
+                        </div>
+
+                        <div class="list-item">
+                            <input id="item-3" type="checkbox" name="checkbox-input">
+                            <label for="item-2">item 1</label>
```
Comment (Rohanmond) [inline document.html:666]: here put item-3 inplace of item-2

**Review by Rohanmond — APPROVED** [review]
just make those changes..you r good to go

**Review by varnita619 — APPROVED** [review]
Looks good to go

**Pull request merged**